README.md

```diff
@@ -115,7 +115,25 @@ If you answer any of the questions below with yes, then Bastion is just for you:
 * Do I have some trust issues with orchestration systems?
 * Do I want to implement my own application lifecycle?
 
-### Get Started
+## Bastion Ecosystem
```

Comment:
Maybe we could add a short introduction here, which would mention that these are building blocks of bastion, and newcomers don't need to understand how they all work (so they dont get too scared ?)